### PR TITLE
Menu and ContextMenu scrolling: fix accessibility issues

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js [hunk 1]
```diff
@@ -3856,6 +3858,87 @@ QUnit.module('adaptivity: behavior', {
     });
 });
 
+let helper;
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js]: Don't u think it's better to declare it in `before` to not use a global scope?

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js [hunk 2]
```diff
@@ -3856,6 +3858,87 @@ QUnit.module('adaptivity: behavior', {
...
+            showFirstSubmenuMode: 'onClick',
+            showSubmenuMode: { name: 'onHover', delay: 0 },
+        });
+        const $item1 = $(menu.element).find(`.${DX_MENU_ITEM_CLASS}`).eq(0);
+
+        $item1.trigger('dxclick');
+
+        const submenu = getSubMenuInstance($item1);
+
+        const $menuItem = $(submenu._overlay.content()).find(`.${DX_MENU_ITEM_CLASS}`).first();
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js]: submenu has `getOverlayContent` method for such purposes
```suggestion
        const $menuItem = submenu.getOverlayContent().find(`.${DX_MENU_ITEM_CLASS}`).first();
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js [hunk 3]
```diff
@@ -3856,6 +3858,87 @@ QUnit.module('adaptivity: behavior', {
...
+        $(submenu.itemsContainer()).trigger($.Event('dxhoverstart', { target: $menuItem.get(0) }));
+        $menuItem.trigger('dxpointermove');
+        this.clock.tick(0);
+        const $submenu = $(submenu._overlay.content()).find(`.${DX_SUBMENU_CLASS}`).eq(1);
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js]:
```suggestion
        const $secondLevelSubmenu = $(submenu._overlay.content()).find(`.${DX_SUBMENU_CLASS}`).eq(1);
```

File: packages/devextreme/js/ui/context_menu/ui.context_menu.js [hunk 4]
```diff
@@ -643,6 +644,7 @@ class ContextMenu extends MenuBase {
 
     _initScrollable($container) {
         this._createComponent($container, Scrollable, {
+            useKeyboard: false,
```
Comment (ksercs) [inline packages/devextreme/js/ui/context_menu/ui.context_menu.js:647]: What about tests for this?
It also should declare that Axe reports an error is scrollable has tabindex

P.s I think we can just check that scrollable's `useKeyboard` option has a correct value, no need to check tabIndex directly

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**